src/plugins/multiline/index.ts

```diff
@@ -0,0 +1,384 @@
+import { PluginController } from "../PluginController";
+import { Config, configList } from "./config";
+import "./multiline.less";
+import { CollapseMode, unverticalify, verticalify } from "./verticalify";
+import { MathQuillField, MathQuillView } from "components";
+import { DispatchedEvent } from "globals/Calc";
+import { Calc } from "globals/window";
+import {
+  deregisterCustomDispatchOverridingHandler,
+  registerCustomDispatchOverridingHandler,
+} from "utils/listenerHelpers";
+
+function focusmq(mq: MathQuillField | undefined) {
+  mq?.focus();
+}
+
+function getController(mq: MathQuillField) {
+  return mq.__controller;
+}
+
+function mqKeystroke(mq: MathQuillField, key: string) {
+  mq.keystroke(key);
+}
```

Comment:
I think this is already in via intellisense. Since that is merged in, you can rely on those instead of duplicating.

Reply:
Fixed!